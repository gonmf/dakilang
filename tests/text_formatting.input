

abc1("ab'cd").
abc2(A > "def ' def").
abc3(A > 'def \' def').
listing
