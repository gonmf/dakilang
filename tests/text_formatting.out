>
>
> abc1("ab'cd").
> abc2(A > "def ' def").
> abc3(A > 'def \' def').
> listing
0: abc1('ab\'cd').
1: abc2(A > 'def \' def').
2: abc3(A > 'def \' def').

